pygmt/tests/test_basemap.py

```diff
@@ -80,3 +80,25 @@ def test_basemap_aliases():
     fig = Figure()
     fig.basemap(region=[0, 360, -90, 90], projection="W7i", frame=True)
     return fig
+
+
+@check_figures_equal()
+def test_basemap_rose():
+    "Create a map with coast and use basemap to add a rose"
+    fig_ref, fig_test = Figure(), Figure()
+    fig_ref.coast(R=[127.5, 128.5, 26, 27], W="1/0.5p")
+    fig_ref.basemap(Td="jBR+w5c")
+    fig_test.coast(region=[127.5, 128.5, 26, 27], shorelines="1/0.5p")
+    fig_test.basemap(rose="jBR+w5c")
+    return fig_ref, fig_test
+
+
+@check_figures_equal()
+def test_basemap_compass():
+    "Create a map with coast and use basemap to add a compass"
+    fig_ref, fig_test = Figure(), Figure()
+    fig_ref.coast(R=[127.5, 128.5, 26, 27], W="1/0.5p")
```

Comment:
It would be better that the reference image is generated without any complication conversions (e.g., from a list to a "/"-seperated string).
```suggestion
    fig_ref.coast(R="127.5/128.5/26/27", W="1/0.5p")
```